### ISSUE TITLE
can you provide the download link of pre-trained R50 model for Voxceleb 2 corpus.

### ISSUE THREAD

**Issue by lyu-joe** [julik43/Online-Identification-of-New-Speakers#3]
thank you for your quick reply,can you provide the download link of pre-trained R50 model for Voxceleb 2 corpus. And by the way ,Is rounds_train * num_train_files * n_row_tf_record a constant . or in other words , does it matter if the training process I designed does not have rounds,i set new num_train_files = rounds_train * num_train_files and delete the rounds parameter

**Comment by julik43** [comment]
I can provide a temporal link for this model, it will be available for 2 weeks in:
https://drive.google.com/open?id=1dSarI4WjOzJ7Pu3WqGaTWjWIBSdTRAJN

I used rounds train to create TFrecords and delete them while training, this kind of files require a lot of capacity, so to use just what is need it, I created the number of training files (num_train_files) with the amount of examples or rows configured in n_row_tf_record and I did this as many round as rounds_train has, and I considered this 1 epoch.
You can change as it best works for you.

I'll close the first issue opened, since you already opened this.

**Comment by lyu-joe** [comment]
thank you again,can i use the test_speakers.txt to test the EER of the model ,no changes to training and testing settings between the model you offered and github version ,right.

**Comment by lyu-joe** [comment]
hey，it's me again ,the pre-trained R50 model you offered was generated at epoch 10 ,so does it means that the model generated at epoch10 is the best model in the training process ,finally ,What is the loss_train,acc_train(batch,global) and the  loss_test,acc_test(batch,global) you trained .

**Comment by julik43** [comment]
"thank you again,can i use the test_speakers.txt to test the EER of the model ,no changes to training and testing settings between the model you offered and github version ,right."

Yes you can use test_speakers.txt to test the EER of the model, and there are no changes between the model offered and the one in github.

"the pre-trained R50 model you offered was generated at epoch 10 ,so does it means that the model generated at epoch10 is the best model in the training process ,finally ,What is the loss_train,acc_train(batch,global) and the loss_test,acc_test(batch,global) you trained"

Epoch 10 was the best of 10 epochs, however the loss of the model was not going significantly down anymore at that epoch. About the accuracy and loss:

Train:
Accuracy: 0.68716373 or 68%
Loss: 0.67271101

Test:
Accuracy: 0.66012500 or 66%
Loss: 0.70348042
EER: 0.33692647058823527 or 33%

This model didn't work well for me, it is consistent, since the train and test accuracy and loss are similar, but still has very poor results, maybe because the size of the input was too small, or because of the modifications I did to the network. In the original ResNet they use an conv2d with stride of 2 at the beginning of each block, I used a conv2d with a stride of 1. Also they use an average pooling before the fully connected layer, but this is along the whole axis, leaving as result an array of 1 x last_number_of_filters, I used and average pooling with a kernel size of 7 and a stride of 2.

**Comment by julik43** [comment]
I'll close this issue, since it was already solved.
